<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="PercentageComplete">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 74, 74);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>102</width>
      <height>102</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:50px;
 border-color: black;
 max-width:100px;
 max-height:100px;
 min-width:100px;
 min-height:100px;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(39, 122, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="YourTurnLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your turn!</string>
    </property>
   </widget>
   <widget class="QLabel" name="MemorizeLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Memorize this!</string>
    </property>
   </widget>
   <widget class="QLabel" name="GameOverLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GAME OVER</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Round</string>
    </property>
   </widget>
   <widget class="QLabel" name="roundNumber">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QListWidget" name="highScoresView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>redButton</zorder>
   <zorder>PercentageComplete</zorder>
   <zorder>blueButton</zorder>
   <zorder>YourTurnLabel</zorder>
   <zorder>MemorizeLabel</zorder>
   <zorder>GameOverLabel</zorder>
   <zorder>startButton</zorder>
   <zorder>label</zorder>
   <zorder>roundNumber</zorder>
   <zorder>highScoresView</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimon">
    <property name="title">
     <string>Simon</string>
    </property>
   </widget>
   <addaction name="menuSimon"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
